python/paddle/fluid/tests/book/high-level-api/recognize_digits/test_recognize_digits_conv.py

```diff
@@ -66,7 +66,8 @@ def train(use_cuda, save_dirname):
         train_func=train_program,
         infer_func=inference_program,
         place=place,
-        optimizer=optimizer)
+        optimizer=optimizer,
+        parallel=True)
```

Comment:
Move parallel to trainer's constructor. It cannot be changed after we create a trainer.